public/script.js

```diff
@@ -0,0 +1,56 @@
+const sourceTextContainer = document.querySelector('.source-text-container')
+
+const spanArray = []
+
+const h1 = document.querySelector('h1')
+
+function populateSourceText(data) {
+  sourceTextContainer.innerHTML = ''
+  spanArray.splice(0, spanArray.length)
```

Comment:
i'm finding it really hard to wrap my head around what this does

Reply:
it turns each word in the source text into a span element (so we can change each of their colours and tab through word by word)

Reply:
actually I was wrong.... What it actually does is it empties the span array and source text container. It's not needed for now but future-proofing in case of adding a reset button, so we know the function works from a clean slate